composer.json

```diff
@@ -15,7 +15,7 @@
         "sensiolabs/security-checker": "^4.1",
         "symfony/asset": "^4.1",
         "symfony/expression-language": "^4.1",
-        "symfony/flex": "^1.0.86",
+        "symfony/flex": "^1.1",
```

Comment:
You have to borrow also the extra config to benefit from v1.1

Reply:
Could you elaborate on this? Which extra config?

Reply:
https://github.com/symfony/demo/pull/854/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R83

Reply:
Thanks, both suggestions have been applied.